src/utils/xhr-loader.js

```diff
@@ -81,8 +81,10 @@ class XhrLoader {
   }
 
   readystatechange (event) {
-    let xhr = event.currentTarget,
-      readyState = xhr.readyState,
+    //On production build, event.currentTarget is null, error "cannot read property 'readyState' of null"
+    //Must use this.loader on production build
+    let xhr = event.currentTarget || this.loader,
+      readyState = xhr.readyState || this.loader.readyState,
```

Comment:
We can simply assign `this.loader` to `xhr` here. The rest does not need to change.

```suggestion
    let xhr = this.loader,
      readyState = xhr.readyState,
```

Reply:
@tssghost89 please make the suggested change above and rebase.